src/miniq.js

```diff
@@ -0,0 +1,15 @@
+(function () {
+  'use strict';
+
+  Promise.delay = (ms) => new Promise((resolve) => setTimeout(resolve, ms))
```

Comment:
Noting that setTimeout is not part of the EcmaScript standard. Nor is it part of the proposed Frozen Realm primordials. Both it and your Promise.delay enables one to measure duration. Which is a problem we've already got. We should at least note these issues in a comment here.